FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,51 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = self.context
+		context.perform {
```

Comment:
During the course this way of having a local reference to the context to avoid capturing self inside of the `context.perform` block is taught. 
The same can be achieved using capture list. 
It saves one line and one variable.

No need to make this amendment, this is just for your interest.

```suggestion
		context.perform { [context] in
```